lib/iris/_lazy_data.py

```diff
@@ -73,6 +73,31 @@ def as_lazy_data(data, chunks=_MAX_CHUNK_SIZE):
     return data
 
 
+def as_concrete_data(data, nans_replacement=None, result_dtype=None):
+    """
+    Return the actual content of a lazy array, as a numpy array.
+
+    If the data is a NumPy ndarray or masked array, return it unchanged.
+
+    If the data is lazy, return the realised result.
+
+    Where lazy data contains NaNs these are translated by filling or conversion
+    to masked data, using the :func:`~iris._lazy_data.convert_nans_array`
+    function.
+    See there for usage of the 'nans_replacement' and 'result_dtype' keys.
```

Comment:
Typo: "there" --> "their".

Alternatively you could change the signature to 

```python
def as_concrete_data(data, **kwargs)
```

as the `kwargs` are not used at all by this function and are just passed straight to `convert_nans_array`. At that point you could replace this description of the `kwargs` with something like

"Kwargs are passed straight to :func:`~iris._lazy_data.convert_nans_array`."

It is a more typical way of doing such things in Python but also less readable...